topdown/graphql.go

```diff
@@ -80,11 +67,110 @@ func validateQuery(schema *gqlast.Schema, query *gqlast.QueryDocument) error {
 		// this affects only the last character(s) in the string.
 		// NOTE(philipc): We know the error location will be in the query string,
 		// because schema validation always happens before this function is called.
-		return fmt.Errorf("%s in GraphQL query string at location %d:%d", strings.TrimSuffix(err[0].Message, "."), err[0].Locations[0].Line, err[0].Locations[0].Column)
+		if err[0].Locations != nil {
+			return fmt.Errorf("%s in GraphQL query string at location %d:%d", strings.TrimSuffix(err[0].Message, "."), err[0].Locations[0].Line, err[0].Locations[0].Column)
+		}
+		return fmt.Errorf("%s in GraphQL query string", strings.TrimSuffix(err[0].Message, "."))
 	}
```

Comment:
[nit] How about something like... 🤔 
```suggestion
		msg := fmt.Sprintf(("%s in GraphQL query string", strings.TrimSuffix(err[0].Message, "."))
		if err[0].Locations != nil {
			msg = fmt.Sprintf("%s at location %d:%d", msg, err[0].Locations[0].Line, err[0].Locations[0].Column)
		}
		return fmt.Errorf(msg)
	}
```

🤷 doesn't matter. 😄